src/components/numbered-code-snippet/__tests__/__snapshots__/numbered-code-snippet.test.js.snap

```diff
@@ -20,12 +20,10 @@ exports[`numbered-code-snippet Basic renders as expected 1`] = `
         data-chunk-code="chunk-0"
       >
         <button
-          className="w-full bg-gray-faint color-gray-dark color-gray-on-hover pl12 py6 cursor-pointer txt-s h30 flex-parent flex-parent--center-cross"
+          className="w-full bg-gray-lighter color-gray-dark color-text-on-hover pl12 py6 cursor-pointer txt-s h30 flex flex--center-cross"
```

Comment:
@katydecorah I chose `color-text` here since `color-gray` is so faint. The lightening of the text on hover is a bit less dramatic, but it seems like the best option.